### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by CyberShadow**
As discussed in #2907, this adds a README file to the project root.

In addition to the links to the website and contributing guide, I took the liberty to add the project logo, and some badges for Travis CI status and issue/PR closure stats.

### CODE REVIEW

**Comment by stapelberg** [comment]
Could you rebase this onto the next branch to pick up commit 3137064efc577e90b342319365b173276b13d236 please?

These steps should do the trick:

```bash
git pull
git rebase next
git push -f git@github.com:CyberShadow/i3 pull-20170909-033500
```

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+![Logo](docs/logo-30.png) i3: A better tiling and dynamic window manager
+========================================================================
+
+[![Build Status](https://travis-ci.org/i3/i3.svg?branch=next)](https://travis-ci.org/i3/i3)
+[![Issue Stats](http://www.issuestats.com/github/i3/i3/badge/issue?style=flat)](http://www.issuestats.com/github/i3/i3)
+[![Issue Stats](http://www.issuestats.com/github/i3/i3/badge/pr?style=flat)](http://www.issuestats.com/github/i3/i3)
```
Comment (Airblader) [inline README.md]: Should this really say "Issue stats" twice?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+i3 is a tiling window manager.
```
Comment (Airblader) [inline README.md]: Let's perhaps add "for X11".

File: README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+![Logo](docs/logo-30.png) i3: A better tiling and dynamic window manager
```
Comment (Airblader) [inline README.md]: Let's shorten this to just "A tiling window manager".

Comment (stapelberg) [reply README.md]: That’d make it inconsistent with our repository description, though.

Comment (Airblader) [reply README.md]: But consistent with the description on the website. :-)

Comment (stapelberg) [reply README.md]: Let’s unify in the other direction, then; changed it in the repo description.

**Comment by CyberShadow** [comment]
> Could you rebase this onto the next branch to pick up commit 3137064 please?

Done.

> Let's shorten this to just "A tiling window manager".

Done.

> Should this really say "Issue stats" twice?

Well, I used the Markdown verbatim as provided by the website, and both badges do link to the same page, but there's no harm in changing it. Done.

> Let's perhaps add "for X11".

Done.
